data/json/monsters/insect_spider.json

```
@@ -2569,7 +2569,7 @@
     "name": { "str": "great mantis", "str_pl": "great mantises" },
     "description": "This ferocious cow-sized mantis stalks the fields of New England.  It's probably capable of killing a moose with its powerful arms.",
     "copy-from": "mon_mantis_giant",
-    "proportional": { "hp": 20, "speed": 0.25, "vision_day": 2 },
+    "proportional": { "hp": 10, "speed": 0.25, "vision_day": 2 },
```

```suggestion
    "proportional": { "hp": 3, "speed": 0.25, "vision_day": 2 },
```
Per the balance doc, this is a better fit for something using blob magic but not an endgame supernatural boss https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/GAME_BALANCE.md